wordpress-mobile/WordPress-iOS
# Fix: Display placeholder text for blank entries in Set Parent view

Fixes #15672

The `titleForDisplay()` already has a placeholder for empty title. This change uses the method to ensure that the same placeholder is reused.

Before | After
-- | --
![15672-before](https://user-images.githubusercontent.com/1299411/121483440-6516d680-c9f8-11eb-924c-b889150ea787.png) | ![15672-after](https://user-images.githubusercontent.com/1299411/121483471-6d6f1180-c9f8-11eb-9bc2-76976b947516.png)

## To test

1. Create a new page with blank title, and publish it.
2. Go to Site Pages, click the three-dots menu on one of the site pages, and tap Set Parent.
3. Verify that the pages with blank title are displayed with placeholder title `(no title)`.

## Regression Notes
1. Potential unintended areas of impact
None.

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Change occurs in `ParentPageSettingsViewController`. This view controller is only invoked by `PageListViewController`.

3. What automated tests I added (or what prevented me from doing so)
The change is applied on `Row`, a private struct inside `ParentPageSettingsViewController` which makes it non-readable from the test module.

PR submission checklist:
